### PR TITLE
Directly return field value without appending typeName when field value is qualified.

### PR DESCRIPTION
**Pull request by CharlesZ-Chen**
Fix issue: https://github.com/typetools/annotation-tools/issues/124

This issue cause me failed to add testing framework for Ontology, as the insertion result cannot compile.

### CODE REVIEW

File: scene-lib/src/annotations/field/EnumAFT.java [hunk 1]
```diff
@@ -46,7 +46,13 @@ public  String toString() {
      */
     @Override
     public String format(Object o) {
-        return typeName + "." + o.toString();
+        String fieldValue = o.toString();
+        if (fieldValue.contains(".")) {
+            //if fieldValue is a qualified value, then return it without appending the typeName
```
Comment (wmdietl) [inline scene-lib/src/annotations/field/EnumAFT.java]: How about changing this to:
```java
if (!fieldValue.contains(".")) {
  // If fieldValue is not qualified, prepend the typeName
  fieldValue = typeName + "." + fieldValue;
}
return fieldValue;
```

File: scene-lib/src/annotations/field/EnumAFT.java [hunk 2]
```diff
@@ -46,7 +46,12 @@ public  String toString() {
      */
     @Override
     public String format(Object o) {
-        return typeName + "." + o.toString();
+        String fieldValue = o.toString();
+        if (!fieldValue.contains(".")) {
+            // If fieldValue is not qualified, prepend the typeName
+            return typeName + fieldValue;
```
Comment (topnessman) [inline scene-lib/src/annotations/field/EnumAFT.java]: Where is the missing "." after typeName?

Comment (CharlesZ-Chen) [inline scene-lib/src/annotations/field/EnumAFT.java]: Oh, thanks for mention that! Good catch, will add it.

**Comment by CharlesZ-Chen** [comment]
PR moved to typetools:

https://github.com/typetools/annotation-tools/pull/137

**Pull request closed without merging**